.travis.yml

```diff
@@ -1,11 +1,16 @@
 language: c
-dist: trusty
+dist: bionic
 sudo: false
 addons:
   apt:
     packages:
       - libffi-dev
       - pkg-config
+      - bison
+      - flex
+      - build-essentials
```

Comment:
was b-e missing in your other PR?

Reply:
Yes and no.
The image from Github Actions already includes all that is necessary from build-essentials. So no need to apt-get install it.
Here I added it "just in case" because I tested locally with a bionic docker image (not the one from Travis) and found I also needed build-essentials... and since I didnt have a way to test Travis builds without pushing a change here, I just added it "just-in-case". Then I discovered the Travis build didnt run because of not enough credits. :) 
Anyway... with Github Actions it all works.